apps/web/server/lib/ssr.ts

```diff
@@ -26,7 +26,10 @@ export async function ssrInit(context: GetServerSidePropsContext) {
   });
 
   // always preload "viewer.public.i18n"
-  await ssr.viewer.public.i18n.fetch();
+
+  const { locale } = await ssr.viewer.public.locale.fetch();
+
+  await ssr.viewer.public.i18n.prefetch({ locale });
```

Comment:
Does it make sense to introduce an i18nWithLocale kind of thing that takes care of both in a single call?